go/vt/vttest/local_cluster.go

```diff
@@ -488,6 +488,43 @@ func (db *LocalCluster) reloadSchemaKeyspace(keyspace string) error {
 	return err
 }
 
+//GetVSchema outputs a JSON structure that contains information about the SrvVSchema
+func (db *LocalCluster) GetVSchema(cell string) (string, error) {
```

Comment:
Rename to GetSrvVSchema, DeleteSrvVSchema etc.

Reply:
Done